.github/CONTRIBUTING.md

```diff
@@ -175,64 +213,56 @@ There are checks that automation will not be able to capture. For example:
    -  Windows 10
       -  Chrome
       -  Microsoft Edge Chromium
-      -  Microsoft Edge UWP (Legacy)
       -  Firefox
-      -  IE11 (except speech features)
+      -  Internet Explorer 11 (except speech features)
    -  Safari on macOS
    -  Safari on iOS or iPadOS
    -  Chrome on Android
--  For accessibility, please refer to [`docs/ACCESSIBILITY.md`](https://github.com/microsoft/BotFramework-WebChat/blob/main/docs/ACCESSIBILITY.md)
-   -  Tab order, content readability, assistive technology-only text, color contrast, etc. must be maintained.
-   -  Assistive technology and browser compatibility
-      -  NVDA/JAWS: Chrome and Firefox
-      -  Narrator: Microsoft Edge family and IE11
-      -  VoiceOver: Safari
-      -  TalkBack: Chrome on Android
--  For internationalization, please refer to [`docs/LOCALIZATION.md`](https://github.com/microsoft/BotFramework-WebChat/tree/main/docs/LOCALIZATION.md)
 -  Feature documentation, samples, live demo, operation of demo bots
    -  All samples must also come with a hosted live demo
    -  Please discuss with us if a specific bot is needed for the live demo
 
-# Workflow
+## Our workflows
 
 Here list how we generally work when [fixing a bug](#fixing-bug) or [implementing a new feature](#implement-new-feature).
 
-## Fixing bug
+### Fixing bugs
 
 Write the bug repro as a test, before fixing the bug.
 
-1. Clone/prepare the repository or reset an existing repository
-   -  To reset the repo, run `npm run tableflip`
-1. Run continuous build, run `npm start`
-1. Convert bug into a test case, under `__tests__/html/this-is-the-bug.html`
-1. Run the test, `npm run start:docker`, then `npm test -- --testPathPattern html/this-is-the-bug.html`
+1. Clone and prepare the repository or reset an existing one
+   -  To reset, run `npm run tableflip`
+1. Run continuous build by running `npm start`
+1. Reproduce the bug on a test page, under `__tests__/html/this-is-the-bug.html`
+1. Run `npm run browser` and navigate to the test page
    -  Make sure the test fail as described in the repro
 1. Fix the bug
-1. Re-run `npm run start:docker` to refresh the code inside Docker
-1. Re-run the test, it should succeed
-   -  Verify screenshots if any
-1. Add a log entry to `CHANGELOG.md`
-1. File a pull request
+1. Reload the test page, it should succeed
+1. Run all test suites in an automated manner
+   -  Run `npm run docker`, followed by `npm test` in a new terminal
+1. Do [final checks](#final-checks)
+1. Submit a pull request
 
-## Implementing new features
+### Implementing new features
 
 Write the user story while implementing the feature.
```

Comment:
docs to developing a user story?

Reply:
Hmmm maybe we can just file an issue for now.

Reply:
#3892